src/org/traccar/model/Position.java

```diff
@@ -93,6 +93,8 @@
     public static final String ALARM_BREAKING = "hardBreaking";
     public static final String ALARM_FATIGUE_DRIVING = "fatigueDriving";
     public static final String ALARM_POWER_CUT = "powerCut";
+    public static final String ALARM_JAMMING = "jamming";
+    public static final String ALARM_TEMPERATURE = "temperature";
```

Comment:
I think it is better to have `LOW_TEMPERATURE` and `HIGH_TEMPERATURE` alarms but it is impossible to determine from packet.

Here is quote from doc:
"The AT+GTTEM command is used to configure temperature alarm of the device. Based on the
working mode set, the device will report temperature alarm when its temperature is outside or
inside of a predefined range."